### PR TITLE
Set up publish

### LINKED ISSUE #25: Make it an installable CLI instead of a repo that you clone
It would be nice to be able to run:

```
npx lighthouse-parade scan ...
```

to get it installed/running instead of cloning the repo.

This requires:
- [x] Adding a CLI entrypoint script that figures out which subcommand is requested
- [x] Adding `bin` to `package.json`
- [ ] Publishing to npm
- [ ] Document changeset/publish workflow

### PR DESCRIPTION
**Pull request by calebeby**
closes #25 

After this PR (if everything goes smoothly):
- Changeset-action will create a release PR for `v0.0.1`
- I'll merge that PR
- Changeset-action will publish to npm and create git tags
- Test published version

### CODE REVIEW

**Comment by calebeby** [comment]
Also @emersonthis if you have a suggestion for a shorter CLI name instead of lighthouse-parade we should probably change that before I merge this

**Comment by emersonthis** [comment]
I think the name is fine. I was overthinking it before.

File: README.md [hunk 1]
```diff
@@ -20,10 +20,7 @@ It is also easy to graph data in this format. The following example is a histogr
 
 ## Installation
 
-1. `nvm use` (if you use NVM)
-2. `npm ci`
-3. `npm run build`
-4. `npm link`
+`npm i -g lighthouse-parade`
```
Comment (emersonthis) [inline README.md:23]: @calebeby We should think about ways to address node version requirement. But I'll create another issue to deal with that.

**Review by emersonthis — APPROVED** [review]

**Comment by emersonthis** [comment]
@calebeby Looks like this dumb update to the README broke the CI somehow: https://github.com/cloudfour/lighthouse-parade/commit/de4ddcfda471110a4637a306f83bc3ec8a25f07e
I'm hoping this PR will fix it... ;-)

**Pull request merged**